src/main/java/org/sagebionetworks/bridge/validators/Schedule2Validator.java

```diff
@@ -30,47 +54,83 @@ public void validate(Object object, Errors errors) {
         Schedule2 schedule = (Schedule2) object;
 
         if (isBlank(schedule.getName())) {
-            errors.rejectValue("name", CANNOT_BE_BLANK);
+            errors.rejectValue(NAME_FIELD, CANNOT_BE_BLANK);
         }
         if (isBlank(schedule.getOwnerId())) {
-            errors.rejectValue("ownerId", CANNOT_BE_BLANK);
+            errors.rejectValue(OWNER_ID_FIELD, CANNOT_BE_BLANK);
         }
         if (isBlank(schedule.getAppId())) {
-            errors.rejectValue("appId", CANNOT_BE_BLANK);
+            errors.rejectValue(APP_ID_FIELD, CANNOT_BE_BLANK);
         }
         if (isBlank(schedule.getGuid())) {
-            errors.rejectValue("guid", CANNOT_BE_BLANK);
+            errors.rejectValue(GUID_FIELD, CANNOT_BE_BLANK);
         }
-        validateFixedLengthLongPeriod(errors, schedule.getDuration(), "duration", true);
+        validateFixedLengthLongPeriod(errors, schedule.getDuration(), DURATION_FIELD, true);
         if (periodInDays(schedule.getDuration()) > FIVE_YEARS_IN_DAYS) {
-            errors.rejectValue("duration", CANNOT_BE_LONGER_THAN_FIVE_YEARS);
+            errors.rejectValue(DURATION_FIELD, CANNOT_BE_LONGER_THAN_FIVE_YEARS);
         }
         if (schedule.getCreatedOn() == null) {
-            errors.rejectValue("createdOn", CANNOT_BE_NULL);
+            errors.rejectValue(CREATED_ON_FIELD, CANNOT_BE_NULL);
         }
         if (schedule.getModifiedOn() == null) {
-            errors.rejectValue("modifiedOn", CANNOT_BE_NULL);
+            errors.rejectValue(MODIFIED_ON_FIELD, CANNOT_BE_NULL);
+        }
+        Set<String> burstIds = new HashSet<>();
+        for (int i = 0; i < schedule.getStudyBursts().size(); i++) {
+            errors.pushNestedPath(STUDY_BURSTS_FIELD+"[" + i + "]");
+            StudyBurst burst = schedule.getStudyBursts().get(i);
+            
+            if (isBlank(burst.getIdentifier())) {
+                errors.rejectValue(IDENTIFIER_FIELD, CANNOT_BE_BLANK);   
+            } else if (burstIds.contains(burst.getIdentifier())) {
+                errors.rejectValue(IDENTIFIER_FIELD, CANNOT_BE_DUPLICATE);
+            }
```

Comment:
What rules should hold for the content of an identifier? I think it's okay to include spaces in events, but would need to retrofit in other places, otherwise, use the same identifier pattern to validate.